tensorbay/client/gas.py

```diff
@@ -345,6 +345,25 @@ def list_dataset_names(self) -> PagingList[str]:
             lambda offset, limit: self._generate_dataset_names(None, offset, limit), 128
         )
 
+    def update_dataset(
+        self,
+        name: str,
+        *,
+        new_alias: str = ...,  # type: ignore[assignment]
+    ) -> None:
+        """Update a TensorBay Dataset.
+
+        Arguments:
+            name: Name of the dataset, unique for a user.
+            new_alias: New alias of the dataset.
+
+        """
+        dataset_id = self._get_dataset(name)["id"]
+        patch_data = {"name": name}
```

Comment:
`update_dataset` is not used to dataset rename, should not put `name` in `patch_data`.